web/src/main/java/com/bazaarvoice/emodb/web/auth/DefaultRoles.java

```diff
@@ -39,6 +39,13 @@
             Permissions.createSorTable(NON_SYSTEM_TABLE),
             Permissions.setSorTableAttributes(NON_SYSTEM_TABLE)),
 
+    // sor_standard with excluding pii tables/placements.
+    sor_standard_without_pii (
```

Comment:
Maybe add a comment explaining why we're adding this, since Emo is open source and visible to anyone.  Without context it may be confusing why we have special roles for pii tables when they aren't referenced anywhere else.